rs/nervous_system/integration_tests/tests/advance_target_version.rs

```diff
@@ -173,9 +174,26 @@ async fn test_get_upgrade_journal() {
 
     // Check that the target version is set to the new version.
     {
-        let sns_pb::GetUpgradeJournalResponse { target_version, .. } =
-            sns::governance::get_upgrade_journal(&pocket_ic, sns.governance.canister_id).await;
+        let sns_pb::GetUpgradeJournalResponse {
+            target_version,
+            upgrade_journal,
+            ..
+        } = sns::governance::get_upgrade_journal(&pocket_ic, sns.governance.canister_id).await;
 
         assert_eq!(target_version, Some(new_sns_version_2.clone()));
+        let upgrade_journal = upgrade_journal.unwrap().entries;
+        let (versions1, versions2) = assert_matches!(&upgrade_journal[..], [
+            sns_pb::UpgradeJournalEntry {timestamp_seconds: Some(_), event: Some(sns_pb::upgrade_journal_entry::Event::UpgradeStepsRefreshed(
+                sns_pb::upgrade_journal_entry::UpgradeStepsRefreshed {upgrade_steps: Some(versions1)}
+            ))}, 
+            sns_pb::UpgradeJournalEntry {timestamp_seconds: Some(_), event: Some(sns_pb::upgrade_journal_entry::Event::UpgradeStepsRefreshed(
+                sns_pb::upgrade_journal_entry::UpgradeStepsRefreshed {upgrade_steps: Some(versions2)}
+            ))}
```

Comment:
Why don't we see any `TargetVersionSet` events in the journal?

Reply:
Because I forgot to add TargetVersionSet events when the test-only API is triggered haha, fixed!